recipes/flatcc/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.12)
```

```suggestion
cmake_minimum_required(VERSION 2.8)
```

too simple for cmake 3+

---

Ok, but the problem is that I never test with cmake 2.8. So yes, that cmake file will probably work with 2.8, but I have no evidence...

---

keep cmake 2.8, if you don't use an explicit cmake 3 feature, keep 2.8. It is still used.